packages/react-components/components/SectionHeadNew.test.tsx

```diff
@@ -0,0 +1,14 @@
+import SectionHeadNew from '@celo/react-components/components/SectionHeadNew'
```

Comment:
Added a new component so other screens using `<SectionHead />` are not impacted by the new designs. 
The new design don't work well on some existing screens.